homeassistant/components/alarm_control_panel/nx584.py

```diff
@@ -0,0 +1,104 @@
+"""
+Networx NX584 interface
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Configuration:
+
+To use the Example custom component you will need to add the following to
+your configuration.yaml file.
+
+alarm_control_panel:
+  platform: nx584
+  host: localhost:5007
+
+Variable:
+
+host
+*Optional
+HOST should be a something like "localhost:5007" which is the
+connection information for talking to the pynx584 backend server.
+"""
+import logging
+import requests
+
+from homeassistant.const import (STATE_UNKNOWN, STATE_ALARM_DISARMED,
+                                 STATE_ALARM_ARMED_HOME,
+                                 STATE_ALARM_ARMED_AWAY)
+import homeassistant.components.alarm_control_panel as alarm
+
+REQUIREMENTS = ['pynx584']
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """ Setup nx584. """
+    host = config.get('host', 'localhost:5007')
+
```

Comment:
There's no login, just connectivity. But I can test connectivity in __init__ and log something, yeah.